docs/simplesamlphp-reference-sp-remote.md

```diff
@@ -300,6 +300,9 @@ The following options can be set:
 :   Note that this option also exists in the IdP-hosted metadata.
     The value in the SP-remote metadata overrides the value in the IdP-hosted metadata.
 
+:   Note: if unset or `true`. this setting will enforce a valid signature.
```

Comment:
I don’t think this correcty describes the new behaviour. It is now this:

- true: authnrequest must be signed (and sig will be validated)
- null: authnrequest may be signed, if it is, sig will be validated
- false: authnrequest sig is never checked

Reply:
You are right.. I've updated it.. Better?

Reply:
I would just make the three values and their meanings explicit in the docs for idp-hosted/sp-remote, like my suggestition, so people don't have to combine different fragments to conclude what does what.